.storybook/pages/CoursePlannerEdit/CoursePlannerEdit.tsx

```diff
@@ -473,10 +473,10 @@ export const CoursePlannerEdit = () => {
 
   return (
     <>
-      <Panel className="u-margin-bottom-lg" variant="squared">
+      <Panel className="mb-6" variant="squared">
         <LayoutContainer>
           <PageHeader
-            className="u-margin-bottom-none"
+            className="mb-0"
             description={
               <HorizontalStepper
                 activeIndex={1}
```

Comment:
I can't leave a comment down there, but there's another utility class on line 496:
```
<ButtonGroup className="u-margin-top-xl-mobile">
```

Reply:
@dierat As discussed with @anniehu4 ,we are going to leave this className="u-margin-top-xl-mobile" as is, since we dont have proper tailwind utilities to replace this class, it has margin unset css properties for media screen-width 768px.

Reply:
Awesome, thanks for the explanation!